partner_employee_quantity/views/res_partner_view.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">Partner form with employees</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <page name="internal_notes" position="inside">
+                <group>
```

Comment:
Why the two levels of group?

Reply:
For show these fields to left side of group. You can inherit this module with add some field in right side of group. I think this is beautiful layout.
![Selection_543](https://user-images.githubusercontent.com/24691983/82082502-5507c700-9712-11ea-9d60-aae02bb84639.png)